content/04.results.md

```diff
@@ -98,21 +102,98 @@ This is the revision of the first paragraph of the introduction of CCC:
+We, as human authors, would make a single change by the end of this sentence to avoid repeating the word "complexity": *"..., while CCC increased the complexity of the model ~~by using different degrees of complexity~~ to capture the relationships"*.
+Since a good summary of the performance of all coefficients was already provided, the next two sentences simply describe what the figure shows while keeping a focus on how CCC works.
+It is remarkable how the model mixed the ideas in the last sentences in the original paragraph (lines 4 to 8) to generate three new ones (lines 3 to 5) with the same meaning but in a more concise and clear way.
+The model also produced high-quality revisions for several other paragraphs that would only need minor changes.
+
+
+Other paragraphs in CCC, however, needed more changes before being ready to be incorporated into the manuscript.
+For instance, for some paragraphs, the model generated a revised text that is shorter, more direct and clear.
+However, important details were removed, and sometimes sentences changed the meaning.
+In this case, we could accept the simplified sentence structure but add back the missing details.
+
+
+In PhenoPLIER, the model also produced high-quality revisions for most paragraphs, while keeping citations and references to figures, tables and other section of the manuscript in the Manubot/Markdown format.
+In some cases, important details were left out, but they could be easily added back while keeping the better sentence structure suggested.
+Other cases clearly showed the limitations of revising one paragraph at a time without considering the rest of the text.
+An example of this is when we describe our CRISPR screening approach to assess whether top genes in an LV could represent good therapeutic targets:
+
+![
+](images/diffs/results/phenoplier-paragraph-01.svg "Diffs - PhenoPLIER results paragraph 01"){width="100%"}
+
+The model generated a paragraph that has a completely different meaning.
+The revised paragraph describes an experiment that does not exist with a reference to a nonexisting section.
+There is no mention of the CRISPR screen and the gene symbols identified to be associated with the regulation of lipids, which are the key elements in the original text.
+Instead, the model seemed to have focused more on the title and keywords of the manuscript (Table @tbl:manuscripts) that are part of every prompt (Figure @fig:ai_revision).
+It included the idea of a "gene co-expression" analysis (a keyword) to identify "therapeutic targets" (another keyword), and replaced the mention of "sets of genes" in the original text with "clusters of genes" in the revision (closer to the "clustering" keyword).
+Although this is clearly a bad revision, the output of the model also suggests that the original paragraph is maybe too short or disconnected from the rest, and that it should probably be merged with the next one (which describes follow-up and related experiments).
```

Comment:
```suggestion
Although this was a poor model-based revision, the output suggests that the original paragraph may be too short or disconnected from the rest and that it could be merged with the next one (which describes follow-up and related experiments).
Paragraphs that AI-based models struggle to revise may also reveal paragraphs that would challenge human readers.
```